lib/cache/expirations/AbsoluteExpiration.ts

```diff
@@ -0,0 +1,27 @@
+import { Expiration } from './Expiration';
+
+export class AbsoluteExpiration implements Expiration {
+
+  private readonly expirations = new Set<string>();
+
+  constructor(
+    private readonly timeout: number,
+  ) { }
+
+  public add(key: string, clear: (key: string) => unknown): void {
```

Comment:
Maybe `never` is more appropriate than `unknown`? And for `clear` maybe `clearCallback` is more suggestive.